Swift/Direct integration/Direct integration/Banner & MPU/BannerViewController.swift

```diff
@@ -1,85 +1,82 @@
 //
-//  ViewController.swift
-//  ThumbnailExample
+//  BannerManager.swift
+//  Direct Integration Sample
 //
-//  Copyright © 2020 Ogury Co. All rights reserved.
+//  Created by Fernand Peng on 09/11/2021.
 //
 
 import UIKit
 import OguryChoiceManager
 import OguryAds
 
-class ViewController: UIViewController {
+class BannerViewController: UIViewController {
     
     @IBOutlet weak var statusTextView: UITextView!
     @IBOutlet weak var mpuView: UIView!
     @IBOutlet weak var smallBannerView: UIView!
-    
-    var mpuLoaded: Bool = false
-    var smallBannerLoaded: Bool = false
-    
+
     var smallBanner: OguryAdsBanner?
     var mpuBanner: OguryAdsBanner?
     
     override func viewDidLoad() {
         super.viewDidLoad()
-        self.addNewStatus("Choice Manager Loading...")
+        self.addNewStatus("[Choice Manager] Loading...")
         
         //The setup of Ogury Choice Manager and Ogury Ads is done AppDelegate.swift file.
         OguryChoiceManager.shared().ask(with: self) { (error, answer) in
```

Comment:
suggestion (no-blocking) : put Choice Manager on a different file

Reply:
I think it's better to keep it in every ViewController so all of them have the whole logic complete example. (beside setup is done in AppDelegate and Constant store in separated file)